index.html

```diff
@@ -1,10 +1,10 @@
 <!doctype html>
 <meta charset='utf-8'>
 <link rel="shortcut icon" href="favicon.ico">
-<link href='dist/dragula.css' rel='stylesheet' type='text/css' />
+<link href='dist/draggle.css' rel='stylesheet' type='text/css' />
 <link href='example/example.css' rel='stylesheet' type='text/css' />
 <title>bpmn-io/draggle</title>
-<h1><a href='https://github.com/bpmn-io/draggle'><img src='resources/logo.svg' onerror='this.src="resources/logo.png"' alt='dragula logo'/></a></h1>
+<h1><a href='https://github.com/bpmn-io/draggle'><img src='resources/logo.png' alt='draggle logo'/></a></h1>
 <h3 class='tagline'><span class='tagline-text'>Drag and drop so simple it hurts</span></h3>
 <div class='examples'>
   <script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?zoneid=1673&serve=C6AILKT&placement=githubcombpmn-io" id="_carbonads_js"></script>
```

Comment:
I think we should remove this, right?

<img width="254" alt="image" src="https://github.com/bpmn-io/draggle/assets/9433996/018029bf-d2fd-4257-bf6f-54a42bb61a74">